File: minimal_api_vue_rdlc_richtext/server/FNDSystem.Infrastructure/DataAccess/Repositories/ActivityRepository.cs
using FNDSystem.Core.Domain;
using FNDSystem.Core.Domain.RepositoryContracts;

namespace FNDSystem.Infrastructure.DataAccess.Repositories;
public class ActivityRepository : GenericRepository<Activity>, IActivityRepository
{
    public ActivityRepository(DBMasterContext masterDBContext, DBSlaveContext slaveDBContext) : base(masterDBContext, slaveDBContext)
    {
    }

}

File: minimal_api_vue_rdlc_richtext/client/src/shared/services/repositories/activity.repository.js
import BaseRepository from 'repositories/base/base.repository';

export default class ActivityRepository extends BaseRepository {
  constructor() {
    super('activity');
  }

  // plantNameで一覧を取得する
  list(plantName) {
    return this.client.get(`/list/${plantName}`);
  }
}
